tests/test_evaluator.py

```diff
@@ -83,224 +69,6 @@ def test_evaluate_multiple_entities_to_keep_correct_statistics():
     assert evaluation_result.results[("O", "ANIMAL")] == 1
 
 
-def test_evaluate_multiple_tokens_correct_statistics():
```

Comment:
Separated between the base evaluator class and the TokenEvaluator class, and created a new test_token_evaluator.py file